astropy/io/ascii/tests/test_html.py

```diff
@@ -603,3 +604,34 @@ def test_raw_html_write_clean():
     <td><em>y</em></td>
    </tr>"""
     assert expected in out.getvalue()
+
+def test_write_table_html():
```

Comment:
Also test with a table that has a multi-dimensional column, i.e. each row is a 1-d array.